src/sparseml/pytorch/utils/exporter.py

```diff
@@ -633,6 +634,36 @@ def _save_label_to_class_mapping(
     )
 
 
+def _fold_identity_initializers(model: onnx.ModelProto):
+    # folds any Identity nodes that have a single input (which is an initializer)
+    # and a single output
+    matches = []
+
+    def is_match(node: onnx.NodeProto) -> bool:
+        return (
+            node.op_type == "Identity"
+            and len(node.input) == 1
+            and len(node.output) == 1
+            and any(node.input[0] == init.name for init in model.graph.initializer)
+        )
```

Comment:
Appreciate you abstracting the actual check into a function, makes it super easy to understand the code
<img src="https://media3.giphy.com/media/VhWVAa7rUtT3xKX6Cd/giphy.gif"/>